gallifreyan.js

```diff
@@ -28,6 +30,11 @@ Array.prototype.remove = function(from, to) {   //based on http://ejohn.org/blog
     return this.push.apply(this, rest);
 };
 
+Array.prototype.removeItem = function(item) {
+    var del = this.indexOf(item);

```

Comment:
>  Toggleable buttons shouldn't be a big deal.

No need to do this in this PR, that'll bring more things to consider, like changing buttons' background color to mark that their mode is enabled etc.